modules/ocf_kubernetes/manifests/ingress.pp

```diff
@@ -0,0 +1,33 @@
+class ocf_kubernetes::ingress {
+
+  $kubernetes_workers_ipv4 = lookup('kubernetes::workers_ipv4')
+  $nginx_version = lookup('kubernetes::nginx_version')
+
+  file {
+    default:
+      require =>  Package['kubeadm', 'kubectl'];
+
+    '/etc/kubernetes/manifests/ingress':
+      ensure => directory,
+      mode   => '0700';
+
+    '/etc/kubernetes/manifests/ingress/ingress-expose.yaml':
+      content => template('ocf_kubernetes/ingress/ingress_expose.yaml.erb'),
+      mode    => '0644';
+  }
+
+  # Add ingress-nginx to the cluster
+  exec { 'init-ingress':
+    command => "kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-${nginx_version}/deploy/mandatory.yaml",
```

Comment:
for clarity perhaps split this into 
```
command => 'kubectl apply -f ${ingress_nginx_url}`
```
and define `$ingress_nginx_url` below `$nginx_version`?